### PR TITLE
Fix the judgment of update flag

### PR DESCRIPTION
**Pull request by wavesZh**
// Describe your PR here; eg. Fixes #issueNo

Fix the judgment of update flag

### CODE REVIEW

**Comment by yu199195** [comment]
pls check ci pass

**Comment by yu199195** [comment]
Hi， can you fix ci error @wavesZh

**Comment by yu199195** [comment]
this is pr maybe Conflicting ,can you update master ? and new pr?

File: shenyu-sync-data-center/shenyu-sync-data-http/src/main/java/org/apache/shenyu/sync/data/http/refresh/DataRefreshFactory.java [hunk 1]
```diff
@@ -57,9 +57,9 @@ public DataRefreshFactory(final PluginDataSubscriber pluginDataSubscriber,
      * @return the boolean
      */
     public boolean executor(final JsonObject data) {
-        final boolean[] success = {false};
-        ENUM_MAP.values().parallelStream().forEach(dataRefresh -> success[0] = dataRefresh.refresh(data));
-        return success[0];
+        return ENUM_MAP.values().parallelStream()
+                .map(dataRefresh -> dataRefresh.refresh(data))
+                .anyMatch(Boolean.TRUE::equals);
```
Comment (dengliming) [inline shenyu-sync-data-center/shenyu-sync-data-http/src/main/java/org/apache/shenyu/sync/data/http/refresh/DataRefreshFactory.java:62]: This causes some values of `ENUM_MAP` to return without executing the `refresh ` method. I don't think we should take this PR.

**Review by dengliming — APPROVED** [review]

**Pull request merged**